lib/spack/spack/package.py

```diff
@@ -918,6 +916,11 @@ def install_configure_args_path(self):
 
         return os.path.join(install_path, _spack_configure_argsfile)
 
+    @property
+    def install_test_root(self):
+        """Return the install test root directory."""
+        return spack.store.layout.metadata_path(self.spec)
```

Comment:
`spack.store.layout.metadata_path(self.spec)` is used in several places. Can we either inline this or rename the property to `metadata_dir` so it can be used in other places that need the same directory?